source/dagger/core/graphics/text.cpp

```diff
@@ -0,0 +1,51 @@
+#include "text.h"
+
+#include "core/core.h"
+#include "core/engine.h"
+#include "core/graphics/sprite.h"
+
+using namespace dagger;
+
+void Text::Set(String font, String message_, Vector3 pos_)
+{
+	this->font = font;
+
+	auto& registry = Engine::Registry();
+
+	assert(Engine::Res<Texture>().contains(fmt::format("spritesheets:{}", font)));
+
+	if (entities.size() > 0)
```

Comment:
where do u set these?

Reply:
we might need to do this removal differently (or add some warning if text is set in the middle of the frame)

Reply:
i thought we were doing a helper for this adding and removing, so decided to just do it the old way and then move it once we establish how we're going to work it in. should i change it right now?

Reply:
also, i set them up in the `Set` function!

Reply:
Also, I found this: https://github.com/skypjack/entt/issues/98

It seems that we are somehow hitting some behaviour which isn't really dangerous?